src/app/view/components/_common/dot-workflows-actions-selector-field/services/dot-workflows-actions.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { CoreWebService } from 'dotcms-js';
+import { pluck } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+import { RequestMethod } from '@angular/http';
+
+export interface DotWorkflowsAction {
+    assignable: boolean;
+    commentable: boolean;
+    condition: string;
+    icon: string;
+    id: string;
+    name: string;
+    nextAssign: string;
+    nextStep: string;
+    nextStepCurrentStep: boolean;
+    order: number;
+    owner?: string;
+    roleHierarchyForAssign: boolean;
+    schemeId: string;
+    showOn: string[];
+}
+
+@Injectable()
+export class DotWorkflowsActionsService {
+    constructor(private coreWebService: CoreWebService) {}
+
+    /**
+     * Get the actions of the received schemas
+     *
+     * @param {string[]} [workflows=[]]
+     * @returns {Observable<DotWorkflowsAction>}
```

Comment:
remove braces

Reply:
Document this extension default generated.